server/website/fabfile.py

```diff
@@ -65,7 +65,7 @@ def status_rabbitmq():
 def start_celery():
     if status_rabbitmq() == STATUS.STOPPED:
         start_rabbitmq()
-    local('python manage.py celery worker --detach --loglevel=info --pool=threads')
+    local('python manage.py celery worker --loglevel=info --pool=threads')
```

Comment:
detach is better,  or you may want to use background task and output to a log file.

Reply:
I don't know why but celery worker cannot be started using detach flag.